plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/FeatureDiagramEditor.java

```diff
@@ -1157,7 +1157,9 @@ public void menuAboutToShow(IMenuManager manager) {
 				selectedAction.setChecked(true);
 				menuManager.insert(1, new Separator("ManualLayoutSeparator"));
 				menuManager.insertBefore("ManualLayoutSeparator", autoLayoutConstraintAction);
-				autoLayoutConstraintAction.setEnabled(!graphicalFeatureModel.getLayout().hasFeaturesAutoLayout());
+				final boolean haveAutoLayout = !graphicalFeatureModel.getLayout().hasFeaturesAutoLayout();
```

Comment:
Weird variable name as haveAutolayout = !hasFeaturesAutoLayout.
Rename to hasConstraintsAutoLayout (if accurate)?